apps/src/sites/studio/pages/levels-music-main.js

```diff
@@ -3,17 +3,17 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import {Provider} from 'react-redux';
 import {getStore} from '@cdo/apps/redux';
-import LabContainer from '@cdo/apps/code-studio/components/LabContainer';
-import MusicLabView from '@cdo/apps/music/views/MusicView';
+import {getStandaloneProjectId} from '@cdo/apps/labs/projects/utils';
+import Lab2Wrapper from '@cdo/apps/code-studio/components/Lab2Wrapper';
 import ProjectContainer from '@cdo/apps/labs/projects/ProjectContainer';
 import ProgressContainer from '@cdo/apps/labs/progress/ProgressContainer';
+import MusicLabView from '@cdo/apps/music/views/MusicView';
 import {logError} from '@cdo/apps/music/utils/MusicMetrics';
-import {getStandaloneProjectId} from '@cdo/apps/labs/projects/utils';
 
 $(document).ready(function () {
   ReactDOM.render(
     <Provider store={getStore()}>
```

Comment:
Can we also replace this with `<Lab2>`? Or are we not even hitting this entrypoint anymore?

Reply:
We still hit this entrypoint when viewing an individual level using a `/levels/[level_id]` URL.